### ISSUE TITLE
Arbitrary Angle Markers

### ISSUE THREAD

**Issue by catthou** [frodare/FlightHud#13]
Hi, great mod, I just wish I could tag 45 degrees and -45 degrees, since those are close to ideal angles for near-infinite flight. If you're feeling very spicy, the ability to add any degree marker I want via config file could be useful for flights between known points in peoples' worlds.

**Comment by frodare** [comment]
Thanks! If I remember correctly, you should able to set that by overriding these in the config:
`pitchLadder_optimumGlideAngle`
`pitchLadder_optimumClimbAngle`

**Comment by catthou** [comment]
That's true! I guess my thought was, there's quite a few different kinds of lines on the pitch ladder, so it'd be neat to be able to declare a line style (short & dashed, long & solid, folded tips...) and an angle, and have it be added to the ladder. That'd probably require all the pitchladder variables be turned into arrays of their own, though, and may be too complex to implement.
